admin-console/src/main/java/com/agiletec/apsadmin/global-messages_en.properties

```diff
@@ -414,6 +414,7 @@ requiredstring = ${getText(fieldName)} is mandatory.
 wrongCharacters=${getText(fieldName)} contains invalid characters, use only alphanumeric characters.
 wrongMaxLength=${getText(fieldName)} is longer than ${maxLength} characters.
 
+#FIXME cancellare ?
```

Comment:
@eugeniosant @zonia3000 can i remove this or it's not safe?

Reply:
@entgigi Yes

Reply:
Already done